src/blockchain_db/lmdb/db_lmdb.h

```diff
@@ -272,7 +272,7 @@ class BlockchainLMDB : public BlockchainDB
 
   virtual uint64_t get_tx_unlock_time(const crypto::hash& h) const;
 
-  virtual std::vector<outkey> get_tx_output_data(const crypto::hash& h) const;
+  virtual std::vector<outkey> get_tx_output_data(const crypto::hash& h, cryptonote::transaction &tx) const;
```

Comment:
This could return a boolean for whether torsion was checked instead of the whole transaction

Reply:
I figured since the function name is `get_tx_output_data` and the tx is already being read into a tx variable, makes sense to just return the whole tx. The torsion check seems a confusing thing to add to this function signature that other contexts wouldn't necessarily care about.